README.md

```diff
@@ -121,3 +126,62 @@ Then that means there is a contentlink that is pointint to a file that doesn't e
 ## Syllabus Content
 
 Look inside the content directory. The documentation is composed of a bunch of markdown files (all named `_index.md`) with a lil metadata. Ok, a lot of metadata.
+
+### WINDOWS 10 USERS
+
+Ensure you have a compatible Linux terminal for windows, if not, checkout: https://ubuntu.com/tutorials/ubuntu-on-windows#1-overview
+
+Make sure your global Github details like your username and email is set up correctly. See above for details on setting up
```

Comment:
Line 134 to 141 Should this not also be with the git clone section, not in the windows specific section as it would apply to all users?

Reply:
Hello

I left it under Windows 10 users so that whoever reads it will hopefully understand that the same still applies, even though you are a windows user.  But if that is confusing I shall delete line 134 since it is already at the top.  I have already moved line 137-141 to the top because that does apply to both Windows users and Linux OS users.